### ISSUE TITLE
Can't pull image from private repo gcr.io/heptio-images/velero:latest 

### ISSUE THREAD

**Issue by mwitzenm** [heptio/velero#1214]
**What steps did you take and what happened:**

I don't have access to pull an image for deployment.  It states I need authorization. for image: gcr.io/heptio-images/velero:latest

**What did you expect to happen:**
I expected to be able to pull the image.  I am still able to pull the ark image from the gcr repo.

**The output of the following commands will help us better understand what's going on**:

Success
```
docker pull gcr.io/heptio-images/ark:latest
Trying to pull repository gcr.io/heptio-images/ark ... 
latest: Pulling from gcr.io/heptio-images/ark
cd784148e348: Pull complete 
fc1bcc5dfa51: Pull complete 
be845fffd0a4: Pull complete 
84cfeeaf260e: Pull complete 
Digest: sha256:97e96915c32af1d382bbb1a56d00cf7e53e5f55839f841af25160b8179782c39
Status: Downloaded newer image for gcr.io/heptio-images/ark:latest
```

Failure
```
docker pull gcr.io/heptio-images/velero:latest
Trying to pull repository gcr.io/heptio-images/velero ... 
Pulling repository gcr.io/heptio-images/velero
unauthorized: authentication required
```

**Comment by nzoueidi** [comment]
The velero image is still not used yet. It will be used in the v0.11.0
As now, it is still being referenced as ark (at least for the latest version v0.10.1).

**Comment by hhuang1illumina** [comment]
@nzoueidi When is v0.10.1 coming out? I've installed velero server but the client is not compatible with the velero server. I am running the following command and it's looking for the old `ark.heptio.com` api rather than `velero.io`. How can I fix this? Can I change the config somewhere in the client?

```
[~] velero get backup-locations                                                                                                                                                                                                     

An error occurred: the server could not find the requested resource (get backupstoragelocations.ark.heptio.com)
```

**Comment by nzoueidi** [comment]
@hhuang1illumina the v0.10.1 is already out (https://github.com/heptio/velero/releases). 
 You may want to take a look also at the getting started page : https://heptio.github.io/velero/v0.10.0/get-started.html

If you are still facing the same issue even after picking the right version. Please file up a separated issue.

**Comment by hhuang1illumina** [comment]
@skriss I am using v0.10.1 tarball and you are correct that it's still `ark` binary while everything else in git is changed to `velero`, so nothing is going to work because of this client and server discrepancy.

**Comment by skriss** [comment]
@hhuang1illumina please see all of the sample YAML files in the release tarball, under `config/`.  Also please use the versioned docs: https://heptio.github.io/velero/v0.10.0/

**Comment by hhuang1illumina** [comment]
@skriss Yes, I realize I can still use the config folder to make ark work. But I can't make velero work at this point with everything that is available on the git website. And it's very confusing for new users is all I am saying.

**Comment by skriss** [comment]
We've released v0.11 which completes the rename. Closing this issue as it should be addressed, but please open a new issue if you encounter any further issues.
